packages/lesswrong/server/fmCrosspost/resolvers.ts

```diff
@@ -66,7 +66,7 @@ export const makeCrossSiteRequest = async <RouteName extends ValidatedPostRouteN
   // Assertion is safe because either we got a result or we threw an error or returned
   const json = await result!.json();
   const validatedResponse = route.responseValidator.decode(json);
-  if (isLeft(validatedResponse)) {
+  if (isLeft(validatedResponse) || 'error' in json) {
```

Comment:
@oetherington Do you think there is a better way to do this, like adding something to the response validator